pkg/network/render.go

```diff
@@ -185,9 +213,9 @@ func RenderAdditionalNetworks(conf *netv1.NetworkConfigSpec, manifestDir string)
 		return nil, errors.Errorf("invalid Additional Network Configuration: %v", errs)
 	}
 
-	// render Multus when additional networks is provided
+	// render the CRD when additional networks are provided
```

Comment:
As per our previous discussion, we need to render the CRD whenever Multus is enabled so administrators can create additional networks out-of-band. I suspect that Multus also needs the CRD to exist to prevent errors.

Reply:
I don't think Multus cares. It only looks for the pod annotations, and only then does it try to get the networks using the CRD. So it's not going to block pods that don't use multiple networks.